specification/cosmos-db/data-plane/Microsoft.TablesStorage/preview/2019-02-02/table.json

```diff
@@ -207,13 +230,19 @@
         "tags": [
           "table"
         ],
+        "consumes": [
+          "application/json"
```

Comment:
Need to override application/json;odata=minimalmetadata?

Reply:
Yeah... maybe not essential as I don't know what the service does with the extra odata tag - however the documented header here is just "application/json"